addons/frag/functions/fnc_frago.sqf

```diff
@@ -96,71 +96,71 @@ _fragArcs set [360, 0];
 
 private _doRandom = true;
 if (_isArmed && {!(_objects isEqualTo [])}) then {
-    if (GVAR(ReflectionsEnabled)) then {
+    if (GVAR(reflectionsEnabled)) then {
         [_lastPos, _shellType] call FUNC(doReflections);
     };
     {
         //if (random(1) > 0.5) then {
             private _target = _x;
             if (alive _target) then {
                 private _boundingBox = boundingBox _target;
+
+                private _cubic = 1;
+                for "_i" from 0 to 2 do {
+                    MUL(_cubic,(abs ((_boundingBox select 0) select _i)) + ((_boundingBox select 1) select _i));
+                };
+
+                if (_cubic <= 1) exitWith {};
```

Comment:
Not the best design, but it works since it's nested in a then block and not the forEach block.